tutor/src/models/course/pair-to-lms.js

```diff
@@ -0,0 +1,24 @@
+import {
+  BaseModel, identifiedBy, observable, field, belongsTo, computed, session,
+} from 'shared/model';
+
+@identifiedBy('course/create')
+export default class CoursePair extends BaseModel {
+
+  @observable course;
+  @observable success;
+
+
+  constructor(course) {
+    super();
+    this.course = course;
+  }
+
+  save() {
+    return this; //{ courseId: this.course.id };
```

Comment:
commented code

Reply:
ah thanks. yeah that was an earlier attempt before I decided to just use `this` for interpolating the url